src/_gettsim/parameters/unterhaltsvors.yaml

```diff
@@ -30,25 +30,91 @@ altersgrenze_mindesteinkommen:
   2017-01-01:
     scalar: 12
     reference: Art. 23 G. v. 14.08.2017 BGBl. I S. 3122.
-altersgrenzen:
+altersgrenzen_bezug:
   name:
     de: Altersgrenzen für den Bezug von Unterhaltsvorschuss
     en: age limits for receiving alimony payments
   description:
     de: >-
       Kinder, die das 12. Lebensjahr noch nicht vollendet haben und bei einem
       alleinerziehenden Elternteil leben haben Anspruch auf Unterhaltszahlungen. Zudem
-      können auch Kinder bis zur Vollendung des 18. Lebensjahres Unterhaltsvorschuss
-      bekommen, wenn das Elternteil ein Mindesteinkommen hat.
+      können seit Juli 2017 auch Kinder bis zur Vollendung des 18. Lebensjahres
```

Comment:
I think this is misleading / too implicit now. The parameter should be marked as obsolete after the reform and it needs to be clear from the description that the stuff from mindestunterhalt is used, then.